lib/BridgedRoom.js

```diff
@@ -434,6 +446,85 @@ BridgedRoom.prototype.onGitterMessage = function(message) {
     });
 };
 
+BridgedRoom.prototype.handleGitterCommand = async function(fromUser, message) {
+    const IS_USERID = /^@.+:.+$/; // Lazy user id regex.
+    if (message === "!matrix") {
+        message = "!matrix help";
+    }
+
+    // Determine if the user is an admin.
+    await this._listGitterUsers();
+
+    if (!this._gitterAdminList.has(fromUser.username)) {
+        await this._main.getRelayGhost().send(this._gitterRoomName, "Only admin's may use the !matrix command");
+        return;
+    }
+
+
+    const commandParts = message.substr("!matrix".length).split(" ");
```

Comment:
![image](https://user-images.githubusercontent.com/1342360/63106736-681b1a00-bf7b-11e9-8579-f73ab9c06f7b.png)

I don't think this does what you want it to do.

```suggestion
    const commandParts = message.split(" ");
```

Alternatively:

```
let commandArgs = message.split(" ");
const command = commandArgs.shift().toLowerCase();
```